test/party_client.py

```diff
@@ -434,3 +434,15 @@ def get_pending_surveys_with_originator_party_id(self, party_id, expected_status
     def delete_pending_surveys_with_batch_no(self, batch_no, expected_status=202):
         response = self.client.delete(f"/party-api/v1/pending-surveys/{batch_no}", headers=self.auth_headers)
         self.assertStatus(response, expected_status)
+
+    def delete_password_verification_token(self, party_id, token, expected_status=200):
```

Comment:
We wouldn't usually have a default var in a function when it is single use. It does make me wonder how we are testing 404 failures, but that is for another story I guess